Parsers/ASimFileEvents/ProductParsers/FileEventLinuxSysmonFileCreated.yaml

```diff
@@ -0,0 +1,70 @@
+Parser:
+  Title: ASIM Sysmon/Linux File Created (Event Id 11) File Event Parser
+  Version: '0.1'
+  LastUpdated: July 27, 2021
+Product:
+  Name: Microsoft Linux Sysmon
+Normalization:
+  Schema: FileEvent
+  Version: '0.1.0'
+References:
+- Title: ASIM File Schema
+  Link: https://aka.ms/AzSentinelFileEventDoc
+- Title: ASIM
+  Link: https://aka.ms/AzSentinelNormalization
+Description: |
+  This is a Query Parser that is used to map Microsoft Linux Sysmon Event id 11 (Event) to the Azure Sentinel Information Model FileEvent schema.
+ParserName: vimFileEventLinuxSysmonFileCreated
+ParserQuery: |
+  Syslog
+  | where SyslogMessage has_all ('<Provider Name="Linux-Sysmon"', '<EventID>11</EventID>')
+  | parse SyslogMessage with  *
+    //    '<Event><System><Provider Name="' msgProviderName:string
+    // '" Guid="{ff032593-a8d3-4f13-b0d6-01fc615a0f97}"/><EventID>'msgEventID:int 
+    // '</EventID><Version>2</Version><Level>'msgLevel:int // Could be indication for success/failure
+    // '</Level><Task>11</Task><Opcode>'msgOpcode:int
+    // '</Opcode><Keywords>'msgKeywords:string
+    '</Keywords><TimeCreated SystemTime="'msgTimeCreated:datetime
+    '"/><EventRecordID>'msgEventRecordID:string
+    '</EventRecordID><Correlation/><Execution ProcessID="'msgProcessID:string // what process is this?
```

Comment:
I would avoid parsing what we don't need to parse to be better version agnostic if they change anything in the future.